spec.html

```diff
@@ -130,21 +130,49 @@ <h1>ThrowAggregateError ( _errors_ )</h1>
   </emu-alg>
 </emu-clause>
 
+<emu-clause id="sec-well-known-intrinsic-objects">
+  <h1>Well-Known Intrinsic Objects</h1>
+  <emu-table id="table-7" caption="Well-Known Intrinsic Objects">
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Intrinsic Name
+          </th>
+          <th>
+            Global Name
+          </th>
+          <th>
+            ECMAScript Language Association
+          </th>
+        </tr>
+        <tr>
+          <td>
+            <ins>%AggregateError%</ins>
+          </td>
+          <td>
+            <ins>`AggregateError`</ins>
+          </td>
+          <td>
+            <ins>The `AggregateError` constructor (<emu-xref href="#sec-native-error-types-used-in-this-standard-aggregateerror"></emu-xref>)</ins>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </emu-table>
```

Comment:
AFAIU this table should also include `AggregateError.prototype`

Reply:
I intentionally did it include it; prototype objects no longer need to be indicated explicitly since we have the intrinsic dot syntax. The current ones are all legacy.